README.md

```diff
@@ -4,7 +4,7 @@ The Swift Package Manager is a tool for managing distribution of source code, ai
 
 We’ve designed the system to make it easy to share packages on services like GitHub, but packages are also great for private personal development, sharing code within a team, or at any other granularity.
 
-Swift Package Manager includes a build system that can build for macOS and Linux. Xcode 11 integrates with libSwiftPM to provide support for iOS, watchOS, and tvOS platforms.
+Swift Package Manager includes a build system that can build for macOS and Linux. Xcode 12 integrates with libSwiftPM to provide support for iOS, watchOS, and tvOS platforms.
```

Comment:
Xcode 11 is where integration with SwiftPM was introduced, how about: "Starting with Xcode 11, Xcode integrates..."

Reply:
Implemented at c37d912446f879e0b188bdbde7e3772b1003687a